scripts/__snapshots__/test-react.js.snap

```diff
@@ -3527,131 +3591,298 @@ ReactStatistics {
           ],
           "message": "",
           "name": "Context.Provider",
-          "status": "NORMAL",
+          "status": "INLINED",
         },
       ],
       "message": "",
       "name": "App",
       "status": "ROOT",
     },
   ],
-  "inlinedComponents": 1,
-  "optimizedNestedClosures": 1,
+  "inlinedComponents": 3,
+  "optimizedNestedClosures": 0,
   "optimizedTrees": 1,
 }
 `;
 
-exports[`Test React with JSX input, JSX output Render props Relay QueryRenderer 1`] = `
```

Comment:
It's becoming quite annoying that we get mismatching snapshots in the diffs. Any way to avoid that?

Reply:
Not really, as we improve inlining, we need to update snapshots. It's a pain but not having the snapshots and regressing is going to be more painful.

Reply:
That's not what I mean.

If you look closely, *the title of the snapshot has changed*. So there's no way to compare them at all: they have moved.

I'm not sure what causes it but my guess is we're adding new tests in the middle of the file, and that confuses the differ. Maybe it's time to split `test-react` into many files?

Reply:
That would be the big fix. Let's do that next week when I'm in London :) putting them in their own files will also speed things up nicely.